app/services/tag_moderators/add.rb

```diff
@@ -49,7 +49,8 @@ def add_to_chat_channels(user, tag)
     end
 
     def add_tag_mod_role(user, tag)
-      user.update(email_tag_mod_newsletter: true) unless user.email_tag_mod_newsletter?
+      user.notification_setting.update(email_tag_mod_newsletter: true) unless
```

Comment:
If it has to be broken over two lines one should probably not use the modifier version of `unless`:

```ruby
unless user.notification_setting.email_tag_mod_newsletter?
  user.notification_setting.update(email_tag_mod_newsletter: true)
end